src/ProductFormInterface.ts

```diff
@@ -0,0 +1,7 @@
+import { ProductItem } from "./ProductItem";
+
+export interface ProductFormInterface{
+    show:Boolean,
```

Comment:
**suggestion:** Use the primitive types: `boolean`, `number`, `string` instead of the class types